src/test/scala/org/apache/spark/sql/BebeFunctionsSpec.scala

```diff
@@ -46,6 +49,27 @@ class BebeFunctionsSpec extends FunSpec with SparkSessionTestWrapper with Column
     }
   }
 
+  describe("bebe_chr") {
+    it("returns the ASCII character of a character") {
```

Comment:
the behaviour is the opposite, you provide a number and returns a single character string with the letter. The expression to do it backwards is the Ascii

Reply:
Ah, thanks!